src/shared/components/Thumbnail/Thumbnail.tsx

```diff
@@ -1,18 +1,27 @@
+import { IcSvgImage } from '@shared/icons';
+
 import * as styles from './Thumbnail.css';
 
 interface ThumbnailProps {
   src?: string;
   alt?: string;
   type: 'square_sm' | 'square_md' | 'square_lg';
 }
+
 const Thumbnail = ({ src, alt, type }: ThumbnailProps) => {
   return (
     <div className={styles.container}>
-      <img
-        className={styles.img({ type })}
-        src={src}
-        alt={alt}
-      />
+      {src ? (
+        <img
+          className={styles.img({ type })}
+          src={src}
+          alt={alt}
+        />
+      ) : (
+        <div className={styles.img({ type, hasImage: false })}>
+          <IcSvgImage style={{ width: '3.2rem', height: '3.2rem' }} />
```

Comment:
이거 style로 하지 말구
```suggestion
          <IcSvgImage width="3.2rem" height="3.2rem"/>
```
으로 수정 곡고